packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -22,25 +23,22 @@ import {
  * @public
  */
 export interface ITree extends IChannel {
+	initialize<TRoot extends ImplicitFieldSchema>(config: TreeConfiguration<TRoot>): void;
```

Comment:
Alternative proposal from Craig:

- Remove initialTree from `TreeConfiguration`, expect users to always call `viewWith` (even on new trees)
- Add explicit `initialize(initialTree)` method to `TreeView` which both upgrades the initial schema and sets up the initial tree.

Reply:
This proposal is now partially reflected in this PR. I didn't yet remove `initialTree` because it requires a lot of test refactoring, but `initialize` is now where we want it.